app/controllers/councillor_contributions_controller.rb

```diff
@@ -31,7 +31,7 @@ def add_contributor
     end
   end
 
-  def create
+  def update
```

Comment:
@hisayohorie This isn't really a conventional #update action, but I agree that it's not a conventional #create action now either. Why not make this the #thank_you action and make it accept the data posted from the form in the #add_contributor form?